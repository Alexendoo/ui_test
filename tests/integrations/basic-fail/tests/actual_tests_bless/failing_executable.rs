//@run

fn main() {
    assert_eq!(5, 6);
}
