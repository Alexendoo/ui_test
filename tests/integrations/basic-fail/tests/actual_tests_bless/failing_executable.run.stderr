thread 'main' panicked at tests/actual_tests_bless/failing_executable.rs:4:5:
assertion `left == right` failed
  left: 5
 right: 6
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
